frontend/flashfolio/src/Viewer.js

```diff
@@ -240,10 +247,23 @@ export default function Viewer({ viewMode = "view" }) {
 				<button id="unshuf" onClick={() => shufOn ? unshufFunction() : null}>Unshuffle</button>)
 			}
 			{viewMode === "edit" && <button onClick={addCard}>Add a card</button>}
-			{viewMode === "edit" && <button onClick={deleteCard}>
-				Delete</button>}
+			{viewMode === "edit" && <button onClick={deleteCard}>Delete</button>}
 			{viewMode === "edit" && <button onClick={saveChanges}>Save Changes</button>}
 			{loginState !== null && <button onClick={cloneD}>Clone Deck</button>}
+			
+			{/* Pop up for delete deck */}
+			<Popup trigger={viewMode === "edit" && <button>Delete Deck</button>} position="right center" modal>
```

Comment:
As a TODO for the cleanup PBI, the delete button should be moved somewhere else, as the main button panel is getting pretty crowded right now.